src/query/sql/src/planner/optimizer/rule/rewrite/rewrite_util/mod.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod extract_or_predicates;
```

Comment:
Can we name the `rewrite_util` to `converts`(or other meaningful name)? `util` is not a good name, and everyone prefers to put their `util` codes to it,  it's hard to maintain,

Reply:
Ok, I will narrow down the scope of the util.

Reply:
Directory name `filter_join_util` -> `filter_join`?

Reply:
done